tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -651,6 +651,287 @@ def process(self, raw, identifiers=None, frame_meta=None):
 
         return result
 
+class PersonVehicleDetectionAdapter(Adapter):
```

Comment:
the adapter is voluminous, I think it makes sense to move it in own file

Reply:
I referred to how pose_estimation_3d.py was defined and created a new file named detection_person_vehicle.py. Detection would be the type that this adapter belongs to and the underbar-separted name would be the specific use case of this adapter module.

This will hopefully solve the pylint error as well.

Reply:
ok, cool :)